.github/actions/inductor-xpu-ut-test/action.yml

```diff
@@ -0,0 +1,91 @@
+name: inductor-xpu-ut-test
+
+inputs:
+  ut_suite:
+    required: true
+    type: string
+    default: 'op_example'
+    description: op_example,op_finegrain,op_ut,torch_xpu. Delimiter is comma
+    
+
+runs:
+  using: composite
+  steps:
+    - name: UT Test (${{ inputs.ut_suite }})
+      shell: bash
+      run: |
+        source activate xpu_op_${ZE_AFFINITY_MASK}
+        source /opt/intel/oneapi/compiler/latest/env/vars.sh
+        cd ${{ github.workspace }}
+        mkdir -p ut_log
+        # check param
+        function contains() {
+            contains_status="echo 'Start $2 ...'"
+            {
+              [[ $1 =~ (^|,)$2($|,) ]]
+            } || {
+              echo "[Warning] $2 is not suppotted type! Skipped!"
+              contains_status="continue"
+            }
+        }
+        set -xe
+        for ut_suite in $(echo ${{ inputs.ut_suite }} |sed 's/,/ /g')
+        do
+          contains "op_example,op_finegrain,op_ut,torch_xpu" $ut_suite
+          $contains_status
+          if [[ ${ut_suite} == 'op_example' ]]; then
+            unset PYTORCH_TEST_WITH_SLOW
+            unset PYTORCH_ENABLE_XPU_FALLBACK
+            cd ${{ github.workspace }}
+            cd examples
+            timeout 8000 pytest -v 2>&1 | tee ${{ github.workspace }}/ut_log/XPU_OP_Examples_UT_Test.log
```

Comment:
This method will block test failure return as job status. E.g. if the test failed, the job status will be success anyway. Another method is add post-op log analysis

Reply:
Sure. Does it check with fail case number? It mean if fail case num > expected num, return false; else return success. 
If so, could you provide the expected num for these UT cases. Thanks~